ext/gmp/tests/construct.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Constructor for GMP
+--EXTENSIONS--
+gmp
+--FILE--
+<?php
+var_dump(new GMP);
+var_dump(new GMP(0));
+var_dump(new GMP(123));
+var_dump(new GMP("0xAA"));
+var_dump(new GMP("12", 4));
+try {
+    var_dump(new GMP("12", 999));
+} catch (ValueError $e) {
+    echo $e->getMessage() . "\n";
+}
```

Comment:
Just for the sake of completeness, add a test with a non-numeric string as the number (e.g. ``'hello'``, or ``''``)